common/src/main/java/com/microsoft/identity/common/internal/cache/BaseMsalTokenCacheItem.java

```diff
@@ -0,0 +1,42 @@
+package com.microsoft.identity.common.internal.cache;
+
+
+import com.google.gson.annotations.SerializedName;
+import com.microsoft.identity.common.internal.providers.azureactivedirectory.AzureActiveDirectoryTokenResponse;
+import com.microsoft.identity.common.internal.providers.oauth2.AuthorizationRequest;
+import com.microsoft.identity.common.internal.providers.oauth2.OAuth2Strategy;
+import com.microsoft.identity.common.internal.providers.oauth2.TokenResponse;
+
+/**
+ * Abstract base class for MsalTokenCacheItems.
+ */
+public abstract class BaseMsalTokenCacheItem {
+
+    @SerializedName("client_id")
+    String mClientId;
+
+    @SerializedName("client_info")
+    String mRawClientInfo;
+
+    @SerializedName("ver")
+    String mVersion = "1";
```

Comment:
I actually don't know what the plan is for a version number... would be good for us to document this... if we can determine the purpose.